docs/wip/theme-shadowing.md

```diff
@@ -1,27 +1,28 @@
 # Shadowing
-This documentation will address Shadowing in Gatsby Themes focusing on how we use it in the Store Framework Jamstack (SFJ), you can find the Shadowing concept general explanations and examples in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/).
+This documentation will address Shadowing in Gatsby Themes focusing on how we use it in the Store Framework Jamstack (SFJ). You can find general explanations and examples of the Shadowing concept in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/).
 
 ## Importance of Shadowing
-In [Gatsby documentation](https://www.gatsbyjs.com/docs/conceptual/plugins-themes-and-starters/) we see that _"Gatsby themes allow Gatsby site functionality to be packaged as a standalone product for others (and yourself!) to easily reuse."_, this means that when you use the theme we developed you will be able to use pre-configured functionality, data sourcing, design and everything else we add to our Store Framework Theme.
+In [Gatsby documentation](https://www.gatsbyjs.com/docs/conceptual/plugins-themes-and-starters/) we see that _"Gatsby themes allow Gatsby site functionality to be packaged as a standalone product for others (and yourself!) to easily reuse."_, it means that when you use the theme we developed, you will be able to use pre-configured functionality, data sourcing, design and everything else we add to our theme.
 
-Our theme package already includes a site structure with a handful of pages and React components that are used to build this pages. You can use this structure to guide your site specifics pages and components. In order to be able to change what we made available, you can make use of the concept of Shadowing.
+Our theme packages already includes a site structure with a handful of pages and React components that are used to build those pages. You can use this structure to guide the development of specific pages and components of your site. In order to be able to change what we offer, you can make use of the Shadowing concept.
```

Comment:
Our theme packages includes a site structure with a handful of pages and React components that are used to build those pages. 
 
 Use this structure as a guide for developing specific pages and components of your site — in order to be able to change what we offer, you can make use of the Shadowing concept.